### PR TITLE
Fixed ms_ssim contribution to reconstruction loss

### PR DESCRIPTION
**Pull request by dmnktk**
During model pretraining we are minimizing the reconstruction loss. Reconstruction loss is a combination of l1 and ms_ssim. To use ms_ssim as a loss function, we need to minimize 1-ms_ssim.

Input sample images:
1_in
3_in
4_in

Output sample images (original reconstruction loss, after 100 epochs):
1_out
3_out
4_out

Output sample images (fixed reconstruction loss, after 100 epochs):
1_out_
3_out_
4_out_

### CODE REVIEW

**Comment by by256** [comment]
Good catch. Merged
